### DISCUSSION TITLE
movefocus

### DISCUSSION THREAD

**Discussion by aloispichler** [hyprwm/Hyprland#3255] [Q&A]
Hello,
`bind = $MainMod right, movefocus, r`
focuses the right app, but it cycles: 1 -> 2 -> 3 -> 1 etc.
Can cycling be avoided, so have 1 -> 2 -> 3 -> 3?

**Comment by MightyPlaza** [comment]
`no_focus_fallback` ?

**Reply by aloispichler** [reply]
Yep, thank you, this is what I was looking for. I did not find it, though.
Thanks again!
